### PR TITLE
Sd collaboration updates

### PR DESCRIPTION
**Pull request by homelessbirds**
Removed the discussion board from the collaboration page as that is going away.

### CODE REVIEW

**Comment by homelessbirds** [comment]
@rhetoric101 could you give this a super quick review? No need to go into structure or anything.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 1]
```diff
@@ -16,63 +16,49 @@ import collaboration_discussion_board_new from 'images/collaboration-discussion-
...
-
-## How it works [#collaborate]
-
-We've integrated discussion tools directly into New Relic so you can: 
-
-* **Start discussions in context at the source.** When you comment on a dashboard or an error in New Relic, that comment stays with that dashboard or error.
-* **Create shared knowledge and history.** Collaborating within the New Relic platform allows your team to build a searchable body of shared knowledge.
+You can collaborate with your teammates by integrating discussion tools into New Relic, allowing you to add comments on any page. Our collaboration tools will enable you to identify problems and find solutions in one place, moving the discussion closer to your data and streamlining your workflows, improving efficiency, and reducing MTTR. When you comment on a dashboard or an error in New Relic, that comment stays with that dashboard or error.
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:19]:
```suggestion
You can collaborate with your teammates by integrating discussion tools into New Relic, allowing you to add comments on any page. Our collaboration tools enable you to identify problems and find solutions in one place, moving the discussion closer to your data and streamlining your workflows, improving efficiency, and reducing mean time to resolution (MTTR). When you comment on a dashboard or an error in New Relic, that comment stays with that dashboard or error.
```

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 2]
```diff
@@ -16,63 +16,49 @@ import collaboration_discussion_board_new from 'images/collaboration-discussion-
...
 
 ## Comment on any page [#comment-on-any-page]
 
-On the top navigation, there's a **Share** button and menu on most pages. If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, select **Share** and then **Start a new discussion**. When you comment on a dashboard or an error in New Relic, that comment stays with your data. You don't need to spend time repeating information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
+On the top navigation, most pages have a **Share** button and menu. If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, select **Share** and then **Start a new discussion**. That comment stays with your data when you comment on a dashboard or an error in New Relic. You don't need to repeat information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:23]:
```suggestion
On the top navigation, most pages have a **Share** button and menu. If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, click **Share** and then **Start a new discussion**. That comment stays with your data when you comment on a dashboard or an error in New Relic. You don't need to repeat information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
```

Comment (rhetoric101) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:23]: @homelessbirds what's the difference between "Share to Slack" and "Start New Discussion"? This paragraph doesn't seem to mention the second option.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 3]
```diff
@@ -16,63 +16,49 @@ import collaboration_discussion_board_new from 'images/collaboration-discussion-
...
+Pages with existing discussions will also have a new **Discussions** button in the top navigation that indicates how many different discussion threads exist on the page. If you click **Discussions**, a comment panel will open where you can view and contribute to those discussions. If any discussions are in **Open** status, the button will be green, as depicted in the screenshot above. The green highlighting calls attention to unanswered questions or unresolved issues you can contribute to. 
 
 ## At-mention your teammates and notify by email [#at-mention-your-teammates]
 
 Like other collaboration tools, when you at-mention (@) your teammates in a comment, they'll receive an email notification. 
 
 ## Built-in screenshot sharing [#built-in-screenshots]
 
-Attach screenshots to your comments by clicking our integrated screenshot selection tool. Just click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 
+Attach screenshots to your comments by clicking our integrated screenshot selection tool. Just click the "screenshot" icon, select the portion of the screen you want, and submit your comment.
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:42]: @homelessbirds I'm not wild about "screenshot" icon. Could you either remove the quotation marks or insert an inline image showing what that is? Will this be obvious to users?

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 4]
```diff
@@ -16,63 +16,49 @@ import collaboration_discussion_board_new from 'images/collaboration-discussion-
...
-  src={collaboration_discussion_board_new}
-/>
+Whenever you comment or get mentioned, you'll automatically follow the discussion. This means you'll receive email notifications when new comments are added.
 
-Go to the discussions board to see a complete history of the discussion threads in your organization. On the left nav, the UMI on the Discussions link indicates discussion threads you are following that have unread comments. On the Discussions page, click a discussion thread to jump to the UI where that conversation is happening.
+Stay in the loop with discussions:
 
-We know that your team uses a wide variety of tools to communicate, but when discussions are spread across platforms and teams, it's easy to lose institutional knowledge. You can now use the discussion board to find active discussions as well as search the archive of shared knowledge. Our discussion board helps you find answers to questions and research how issues and problems were solved in the past.
+* Look for the bell icon on the discussion to see if you're following.
+* Want to stop receiving notifications? Unfollow the discussion from the "More Actions" menu.
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:62]:
```suggestion
* Want to stop receiving notifications? Go to the **More Actions** menu and click **Unfollow**.
```

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 5]
```diff
@@ -16,63 +16,49 @@ import collaboration_discussion_board_new from 'images/collaboration-discussion-
...
+* Not interested in a specific discussion? Don't worry; you can easily follow it manually through the same menu.
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:63]: @homelessbirds I'm not really sure what it means to following a discussion manually. What is this bulleted point trying to say?

**Review by rhetoric101 — APPROVED** [review]
@homelessbirds I made some optional suggestions.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx (hunk 3 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:42]: I fixed this. It is obvious to users, so I just removed the quotation marks.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx (hunk 5 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:63]: Good callout! I fixed this.

**Review by rhetoric101 — APPROVED** [review]
@homelessbirds thanks for making these changes!

**Pull request merged**